recipes/proteinortho/meta.yaml

```diff
@@ -20,13 +20,13 @@ requirements:
     - cmake
   host:
     - openmp
-    - lapack
```

Comment:
This package is deprecated. Use `liblapack`.

Reply:
lapack does not work:

mac runtime-error:
`06:46:39 BIOCONDA INFO (ERR) Reason: Incompatible library version: proteinortho_clustering requires version 3.0.0 or later, but liblapack.3.dylib provides version 0.0.0`

But liblapacke works ... idk